src/webui/www/private/preferences_content.html

```diff
@@ -712,6 +725,11 @@
         $('bypass_auth_subnet_whitelist_textarea').setProperty('disabled', !isBypassAuthSubnetWhitelistEnabled);
     };
 
+    updateHostHeaderValidationSettings = function() {
```

Comment:
This should also be called in `loadPreferences()` so that the textarea is disabled on initial load, as appropriate.

Reply:
Done!